java/.settings/JMRI PanelPro.launch

```diff
@@ -20,5 +20,5 @@
 <listEntry value="org.eclipse.debug.ui.launchGroup.run"/>
 </listAttribute>
 <booleanAttribute key="org.eclipse.jdt.launching.ATTR_USE_START_ON_FIRST_THREAD" value="true"/>
-<stringAttribute key="org.eclipse.jdt.launching.WORKING_DIRECTORY" value="${project_loc:jmri}"/>
+<stringAttribute key="org.eclipse.jdt.launching.WORKING_DIRECTORY" value="${project_loc:JMRI}"/>
```

Comment:
This is changed to lowercase by @DanielBoudreau five days ago in commit https://github.com/JMRI/JMRI/commit/d9ac003c29bf7cb2cd6eea665ab48a16af2b34a9 .

Reply:
As per @danielb987 please revert this change